### ISSUE TITLE
Flicker Issue on the adjacent second view while moving from special item to first view.

### ISSUE THREAD

**Issue by rajendrattn** [siralam/LoopingViewPager#26]
Flicker Issue on the adjacent second view while moving from special item to first view. can you please help me to resolve this issue ?

**Comment by siralam** [comment]
I don't get your issue. What do you mean by "adjacent second view" and "special item"?

**Comment by rajendrattn** [comment]
In your sample there is one pager item as a special view if pager automatically moves from left to right then the next page item created immediately. Which looks awkward behavior.

**Comment by siralam** [comment]
Ah I now understand what you mean. If adjacent page peeking is enabled by using setting clipToPadding="false", when user scrolls to first / last page the silent page change will occur only after scroll state is idle, thus user will see the moment that page is being created.

This is indeed an issue but given the current approach of infinite scroll, at the moment I don't have any idea how to avoid it. I will mark it as won't fix first.
